presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-cassandra/health-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eo pipefail
+
+# https://github.com/docker-library/healthcheck/blob/master/cassandra/docker-healthcheck
+host="$(hostname --ip-address || echo '127.0.0.1')"
+
+if cqlsh -u cassandra -p cassandra "$host" < /dev/null; then
+	exit 0
+fi
+
+exit 1
```

Comment:
that's a verbose way of saying

```
[exec] cqlsh -u cassandra -p cassandra "$host" < /dev/null
```

no `if` needed for two reasons:

- `-e` ends scripta after command fail
- i believe bash exits with last command status anyway
  - `exec` is optional, but IMO helps read this